lighthouse-core/report/report-generator.js

```diff
@@ -151,8 +151,10 @@ class ReportGenerator {
       }
 
       const partials = reportPartials.report.partials;
-      Handlebars.registerPartial(audit.name,
-        Handlebars.template(partials[audit.name] || partials['null-formatter']));
+      const partial = partials[audit.name] ||
+          partials[audit.extendedInfo.formatter] ||
```

Comment:
should we try looking at `partials[audit.name]` at all or just keep it to `partials[audit.extendedInfo.formatter] || null`?
If we can, I'd prefer that, as it's more declarative which formatter will be used.